src/components/homepage/RecommendProducts.jsx

```diff
@@ -0,0 +1,73 @@
+import "../../styles/components/RecommendProducts.css";
+import brandArmani from "../../assets/images/armani.jpg";
+import brandGucci from "../../assets/images/gucci.jpg";
+import brandNorthFace from "../../assets/images/northFace.jpg";
+
+import suit from "../../assets/images/suit.jpg";
+import whiteShirt from "../../assets/images/man-white-shirt.jpg";
+import dress from "../../assets/images/dress.jpg";
+import jeans from "../../assets/images/blue-jeans.jpg";
+import tShirt from "../../assets/images/black-t-shirt.jpg";
+

```

Comment:
make a common index.js file  , where you can import all images and you can import here it's a good structure

Reply:
Done a4a3afd394857327a6495d55074fe08a881ab0a7